Makefile

```diff
@@ -5,13 +5,15 @@ BINDIR ?= $(PREFIX)/bin
 LIBDIR ?= $(PREFIX)/lib
 ETCDIR ?= /etc
 AURUTILS_LIB_DIR ?= $(LIBDIR)/$(PROGNM)
+AURUTILS_GIT_VER := $(shell git describe --tags)
```

Comment:
This won't work when building from a tarball, right...

Maybe make it `?=` and in PKGBUILD pass `pkgver` as `make` parameter?

Reply:
Good point

Reply:
https://github.com/AladW/aurutils/pull/788/commits/de351889d62feb30cb2f978b52415448cc1e3097 adds a fallback that doesn't require changes in the PKGBUILD. Thoughts?

Reply:
This I'm fine with, the small downside comparing to passing pkgver in PKGBUILD is that now you need to remember to adjust version before each next release 😉

Reply:
Sure (and I'll probably forget to update it at some point...) but on the upside people installing from a tarball directly will also have a functional `aur --version` by only running `make` 🙂